.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,90 @@
+name: "Bug report"
+description: "Create a report to help us improve TypeScript"
+body:
+  - type: markdown
+    attributes:
+      value: Please fill in each section completely. Thank you!
+  - type: input
+    id: search_terms
+    attributes:
+      label: "🔎 Search Terms"
+      description: |
+        What search terms did you use when trying to find an existing bug report?
+
+        List them here so people in the future can find this one more easily.
+    validations:
+      required: true
+  - type: textarea
+    id: version_info
+    attributes:
+      label: "🕗 Version & Regression Information"
+      description: |
+        When did you start seeing this bug occur?
+
+        \"Bugs\" that have existed in TS for a long time are very likely to be FAQs; refer to
```

Comment:
This doesn't need to be escaped anymore, as far as I'm aware; in fact, all of the escapes should be able to go away.